app/src/androidTest/java/ch/hikemate/app/ui/map/RunHikeScreenTest.kt

```diff
@@ -300,4 +360,159 @@ class RunHikeScreenTest {
         .assertIsDisplayed()
         .assert(hasText("23% complete"))
   }
+
+  @Test
+  fun runHikeScreen_fetchesFacilities() = runTest {
+    val listFacility =
+        listOf(
+            Facility(
+                type = FacilityType.TOILETS, // We'll test the toilets drawable
+                coordinates = LatLong(45.9, 7.6)))
+    `when`(facilitiesRepository.getFacilities(any(), any(), any())).then {
+      val onSuccess = it.getArgument<(List<Facility>) -> Unit>(1)
+      onSuccess(listFacility)
+    }
+    setupCompleteScreenWithSelected(detailedHike2)
+    verify(facilitiesRepository).getFacilities(any(), any(), any())
+  }
+
+  @Test
+  fun runHike_displaysCorrectDrawableForFacilityType() = runBlocking {
```

Comment:
Suggestion: Love the test still, but this one two is just copy paste from the hike details, or? It might also be good to inherit from a common base class.

Reply:
I don't really understand what you mean by "base class"

Reply:
I would like to say yes to this. Unfortunately we won't have the time to do it...

Reply:
Anyways I wouldn't know how you would inherit from two test classes and I also see that most tests that handle the screen repeat themselves so we would have to refactor all those tests

Reply:
Ok I agree, there is no time. Let's leave it, I've also discussed with the group. Since its not a critical issue we can leave it.